src/Temporalio/Bridge/src/worker.rs

```diff
@@ -49,6 +48,43 @@ pub struct WorkerOptions {
     nondeterminism_as_workflow_fail_for_types: ByteArrayRefArray,
 }
 
+#[repr(C)]
+pub struct TunerHolder {
```

Comment:
```suggestion
pub struct WorkerTuner {
```

I think this is a better name, but doesn't matter much to have this separate/different "holder" terminology in the bridge only I suppose so long as user's don't have to see it

Reply:
This matches the thing it's getting turned into Core side so I think it's worth keeping